ass2/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1,  maximum-scale=1.0"/>
+    <title>Second UI-tak</title>
+    <link rel="shortcut icon" href="img/favicon.png">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" type="text/css" href="css/icomoon.css">
+    <link rel="stylesheet" type="text/css" href="fonts/open-sans">
+  </head>
+
+  <body>
+    <div class="navbar-container">
+    <nav class="navbar">
+      <a href="#!" class="brand-logo"><img src="img/logo.png"></a>
+      <div class="menu"><i class="icon-menu"></i></div>
+      <div class="search-container">
+        <form>
+        <label class="icon-search"></label>
+          <input type="text" name="search" placeholder="Lookup User">
```

Comment:
@priyanka0901 This is a search input field so we should use input type `search` instead of `text`